certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+          "type": "string"
+        },
+        "records": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "uuid",
+        "name",
+        "pipeline",
+        "documents",
+        "records"
+      ]
+    },
+    "Document": {
+      "title": "Document",
+      "type": "object",
+      "properties": {
+        "uuid": {
+          "type": "string",
+          "description": "uuid"
+        },
+        "created": {
+          "type": "string",
+          "description": "created"
+        },
+        "modified": {
+          "type": "string",
+          "description": "modified"
+        },
+        "status": {
+          "type": "string",
+          "description": "status"
+        },
+        "archived": {
+          "type": "boolean",
+          "description": "archived"
+        },
+        "removed": {
+          "type": "boolean",
+          "description": "removed"
+        },
+        "file_name": {
+          "type": "string",
+          "description": "file_name"
+        },
+        "page_count": {
+          "type": "integer",
+          "format": "int32",
+          "description": "page_count"
+        },
+        "pages": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "number": {
+                "type": "integer",
+                "format": "int32",
+                "description": "number"
+              },
+              "width": {
+                "type": "integer",
+                "format": "int32",
+                "description": "width"
+              },
+              "height": {
+                "type": "integer",
+                "format": "int32",
+                "description": "height"
+              }
+            }
+          },
+          "description": "pages"
+        },
+        "dataset": {
+          "type": "string",
+          "description": "dataset"
+        },
+        "records": {
+          "type": "array",
+          "items": {},
+          "description": "records"
+        },
+        "pipeline_records": {
+          "type": "object",
+          "properties": {},
+          "description": "pipeline_records"
+        },
+        "_s3_key": {
+          "type": "string",
+          "description": "_s3_key"
+        },
+        "_replaced_by": {
+          "type": "string",
+          "description": "_replaced_by"
+        },
+        "subject": {
+          "type": "string",
+          "description": "subject"
+        },
+        "sender": {
+          "type": "string",
+          "description": "sender"
+        },
+        "folder": {
+          "type": "string",
+          "description": "folder"
+        },
+        "targets": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "targets"
+        },
+        "watermark_removed": {
+          "type": "boolean",
+          "description": "watermark_removed"
```

Comment:
@tori-alkymi All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)